pkg/util/taints/taints.go

```diff
@@ -20,10 +20,22 @@ package taints
 import (
 	"fmt"
 	"strings"
+	"time"
 
+	apierrs "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/validation"
+
 	"k8s.io/kubernetes/pkg/api"
```

Comment:
util packages shouldn't depend on particular API objects.  This looks a lot more like a utility for a given API, which should live somewhere under that API, not the general util package.